src/main/kotlin/no/nav/api/dialog/DialogService.kt

```diff
@@ -1,10 +1,96 @@
 package no.nav.api.dialog
 
+import kotlinx.serialization.Serializable
 import no.nav.api.dialog.saf.SafService
+import no.nav.api.dialog.sf.SFService
+import no.nav.api.pdl.PdlService
+import no.nav.modiapersonoversikt.commondomain.TemagruppeTemaMapping.hentTemagruppeForTema
 
 class DialogService(
-    private val safService: SafService
+    private val safService: SafService,
+    private val sfService: SFService,
+    private val pdlService: PdlService
 ) {
-    fun sendSporsmal() {}
-    fun sendInfomelding() {}
+    @Serializable
+    data class SendInfomeldingRequest(
+        val tekst: String,
+        val tema: String,
+        val enhet: String
+    )
+
+    @Serializable
+    data class SendSporsmalRequest(
+        val tekst: String,
+        val tema: String,
+        val enhet: String
+    )
+
+    @Serializable
+    data class Response(
+        val kjedeId: String,
+    )
```

Comment:
Denne er vel også det samme som `HenvendelseDTO` 🤔 Men vet ikke hva som er best practice her, kanskje vi bare skal beholde det sånn ettersom at `HenvendelseDTO` beskriver respons fra SF, mens denne beskriver response i vårt API.

Reply:
Usikker på hva som er best practice her selv, men det er jo egentlig to forskjellige response - ene er den vi får fra SF den andre er vår. Jeg tenker det er greit å skille på dem slik at man ser at det ikke er samme response selvom den ser lik ut, men kan være jeg tenker feil?

Reply:
Ja vi kan sikkert la det stå sånn 👌🏼